roles/cdi/tasks/version.yml

```diff
@@ -0,0 +1,15 @@
+---
+# 'version_compare' changes to 'version' in ansible-2.5
+- name: "cdi_release_tag >= 1.0 only works with >= kubevirt_version 0.7.0-alpha.4.  Setting cdi_release_tag to 0.5"
+  set_fact: cdi_release_tag="0.5"
+  when: "{{ '1.0' is version_compare(cdi_release_tag, '<=') }}
+  	and {{ '0.7.0-alpha.4' is version_compare(kubevirt_version, '>') }}"
+
+# 'version_compare' changes to 'version' in ansible-2.5
+- name: "kubevirt_version 0.7.0-alpha.4 requires cdi_release_tag >= 1.0.  Setting cdi_release_tag to 1.0"
+  set_fact: cdi_release_tag="1.0"
+  when: "{{ '1.0' is version_compare(cdi_release_tag, '>') }}
```

Comment:
you said cdi 1.0 only work >= 0.7.0-alpha.4. But if cdi == 1.1, kubevirt == 0.7.0-alpha.4 it will still works right ?

so I guess it should be:
```
set_fact: cdi_release_tag="0.5"
when: "{{ '1.0' is version_compare(cdi_release_tag, '>=') }} and {{ '0.7.0-alpha.4' is version_compare(kubevirt_version, '<') }}"
```
no ?

Reply:
I think I need to update the task names.